seed/building_sync/building_sync.py

```diff
@@ -237,534 +154,185 @@ def export(self, property_state, process_struct=ADDRESS_STRUCT):
+                messages['warnings'].append(f'Skipping measure {measure["name"]} due to missing category')
+                continue
+
+            measures.append(measure)
+
+        scenarios = []
+        for scenario in result['scenarios']:
+            # process the scenario meters (aka resource uses)
+            meters = {}
+            for resource_use in scenario['resource_uses']:
+                meter = {}
+                meter['source'] = Meter.BUILDINGSYNC
+                meter['source_id'] = resource_use['source_id']
+                meter['type'] = resource_use['type']
+                meter['units'] = resource_use['units']
+                meter['is_virtual'] = scenario['is_virtual']
+                meter['readings'] = []
+
+                meters[meter['source_id']] = meter
+
+            # process the scenario meter readings
+            for series_data in scenario['time_series']:
+                reading = {}
+                reading['start_time'] = series_data['start_time']
+                reading['end_time'] = series_data['end_time']
+                reading['reading'] = series_data['reading']
+                reading['source_id'] = series_data['source_id']
+                reading['source_unit'] = meters[reading['source_id']].get('units')
+
+                # add reading to the meter
+                meters[reading['source_id']]['readings'].append(reading)
+
+            # create scenario
+            seed_scenario = {}
+            seed_scenario['id'] = scenario['id']
+            seed_scenario['name'] = scenario['name']
+            seed_scenario['reference_case'] = scenario['reference_case']
+            seed_scenario['annual_site_energy_savings'] = scenario['annual_site_energy_savings']
+            seed_scenario['annual_source_energy_savings'] = scenario['annual_source_energy_savings']
+            seed_scenario['annual_cost_savings'] = scenario['annual_cost_savings']
+            seed_scenario['annual_electricity_savings'] = scenario['annual_electricity_savings']
+            seed_scenario['annual_natural_gas_savings'] = scenario['annual_natural_gas_savings']
+            seed_scenario['annual_site_energy'] = scenario['annual_site_energy']
+            seed_scenario['annual_site_energy_use_intensity'] = scenario['annual_site_energy_use_intensity']
+            seed_scenario['annual_source_energy'] = scenario['annual_source_energy']
+            seed_scenario['annual_source_energy_use_intensity'] = scenario['annual_source_energy_use_intensity']
+            seed_scenario['annual_electricity_energy'] = scenario['annual_electricity_energy']
+            seed_scenario['annual_peak_demand'] = scenario['annual_peak_demand']
+            seed_scenario['annual_natural_gas_energy'] = scenario['annual_natural_gas_energy']
+            seed_scenario['measures'] = [id['id'] for id in scenario['measure_ids']]
+
+            seed_scenario['meters'] = list(meters.values())
+            scenarios.append(seed_scenario)
```

Comment:
```suggestion
            scenario['meters'] = list(meters.values())
            scenario['measures'] = [id['id'] for id in scenario['measure_ids']]
            
            # remove key values that are no longer needed
            del scenario['time_series']
            del scenario['resource_uses']
            del scenario['measure_ids']
            del scenario['is_virtual']            
                        
            scenarios.append(scenario)
```

I'm thinking that you could trim the existing scenario dict and add key values as needed rather than construct a new scenario dict. Potentially, it could look like this but definitely let me know if I'm missing something both in general or in the key/values needed in the proposed change.

Reply:
I think your approach is definitely correct and simpler. My thought process was that parsing is already kind of magic and loosey goosey (we're using dicts for standardized structure!) I wanted to be as explicit as possible here about what we expect our incoming data to look like and what our outgoing data should contain, so I might prefer keeping it very explicit